azext_iot/dps_sdk/device_provisioning_service_service_runtime_client.py

```diff
@@ -33,12 +36,12 @@ def __init__(
         if credentials is None:
             raise ValueError("Parameter 'credentials' must not be None.")
         if not base_url:
-            base_url = 'https://<fully-qualified IoT hub domain name>'
+            base_url = 'https://localhost'
 
         super(DeviceProvisioningServiceServiceRuntimeClientConfiguration, self).__init__(base_url)
 
         self.add_user_agent('deviceprovisioningserviceserviceruntimeclient/{}'.format(VERSION))
-        self.add_user_agent('MicrosoftAzure/IoTPlatformCliExtension/{}'.format(extver))
```

Comment:
Please keep the custom user agent for downstream usage metrics

Reply:
change reverted